pom.xml

```diff
@@ -9,9 +9,42 @@
         <version>9</version>
     </parent>
 
-    <groupId>uk.co.qualitycode</groupId>
+    <groupId>io.github.excitedkettle</groupId>
     <artifactId>functional-utils</artifactId>
-    <version>2.0-SNAPSHOT</version>
+    <version>2.0</version>
+    <packaging>jar</packaging>
+
+    <name>Functional Utils</name>
+    <description>A utility library for adding functional interfaces to core java 8+</description>
+
+    <licenses>
+        <license>
+            <name>Apache License, Version 2.0</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+            <distribution>repo</distribution>
+        </license>
```

Comment:
No idea if this is the right license to use, it was just the default one I found online.